code/datums/diseases/advance/symptoms/sensory.dm

```diff
@@ -24,27 +24,36 @@ Bonus
 	level = 5
 	severity = 0
 
+/datum/symptom/sensory_restoration/proc/check_and_add(var/reagent, var/check, var/add, var/mob/living/M)
```

Comment:
No implied var/ - So (reagent, check, add, mob/living/M) instead of (var/reagent, var/check) etc.